var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -1109,3 +1109,21 @@ def detect_gdc(self):
                         ),
                     ),
                 )
+
+    @classmethod
+    def runtime_constraints(cls, *, compiler, pkg):
```

Comment:
Where do you want this documented? Documentation for this function was left out on purpose since:
1. It's too early to call this API final, so not worth advertizing it in docs
2. Having docstring in packages is a lot of duplication, since this will not be called only here.

I think I'll proceed adding docstring here with a note that it'll be deleted at a later point, but let me know if you prefer otherwise.

Reply:
I think that's fine for now.

Reply:
Done